pwndbg/gdblib/disasm/instruction.py

```diff
@@ -248,6 +247,28 @@ def __init__(self, cs_insn: CsInsn | None) -> None:
         None if we don't have a determination (most cases)
         """
 
+        self.declare_is_unconditional_jump: bool = False
+        """
+        This field is used to declare that this instruction is an unconditional jump.
+        Most of the type, we depend on Capstone groups to check for jump instructions,
+        but sometimes these are lacking, such as in the case of general-purpose instructions
+        where the PC is the destination register, such as Arm `add`, `sub`, `ldr`, and `pop` instructions.
+
+        In these cases, we want to forcefully state that this instruction mutates the PC, so we set this attribute to True.
+
+        This helps in two cases:
+        1. Disassembly splits
+        2. Instructions like `stepuntilasm` work better, as they detect these as branches to stop at.
+        """
```

Comment:
Btw I just noticed that we document fields under them? I think we should document them above (just make sure that its not the function docstring 😓)